### ISSUE TITLE
warning with `R 3.6.0`

### ISSUE THREAD

**Issue by IndrajeetPatil** [tidyverse/rvest#227]
With the development version of `R 3.6.0` I am getting the following warning once per session.
Wasn't getting any such warning with `R 3.5.1`, so just wanted to raise this issue before the next release.
```
Registered S3 method overwritten by 'rvest':
  method            from
  read_xml.response xml2
```

**Comment by hadley** [comment]
CRAN does not: https://www.r-project.org/nosvn/R.check/r-devel-linux-x86_64-debian-clang/rvest-00check.html

**Comment by lemna** [comment]
I'm still receiving this message with the released version of R 3.6.0. Similar messages for dplyr and ggplot2 disappeared after upgrading those their latest github versions, but that did not work for rvest.

```r
> library(rvest)
Loading required package: xml2
Registered S3 method overwritten by 'rvest':
  method            from
  read_xml.response xml2
>  packageVersion("rvest")
[1] ‘0.3.3.9000’
> R.version
               _                           
platform       x86_64-w64-mingw32          
arch           x86_64                      
os             mingw32                     
system         x86_64, mingw32             
status                                     
major          3                           
minor          6.0                         
year           2019                        
month          04                          
day            26                          
svn rev        76424                       
language       R                           
version.string R version 3.6.0 (2019-04-26)
nickname       Planting of a Tree 
```
The message is also printed when I call `library(tidyverse)`.
